src/ts/index.ts

```diff
@@ -36,12 +35,60 @@ export { default as Section } from './components/content/section';
 export { default as SideBar } from './components/navigation/side-bar';
 export { default as SpacedGroup } from './components/misc/spaced-group';
 export { default as SpeechBubble } from './components/misc/speech-bubble';
-export { default as Tabs } from './components/tabs/tabs';
 export { default as Tab } from './components/tabs/tab';
 export { default as Table } from './components/tables/table';
 export { default as TableBody } from './components/tables/table-body';
 export { default as TableCell } from './components/tables/table-cell';
 export { default as TableHead } from './components/tables/table-head';
 export { default as TableHeader } from './components/tables/table-header';
 export { default as TableRow } from './components/tables/table-row';
+export { default as Tabs } from './components/tabs/tabs';
 export { default as Well } from './components/misc/well';
+
+export { AlertProps } from './components/misc/alert';
```

Comment:
All of these should be able to be exported on the same line e.g.

```ts
export { default as Alert, AlertProps } from './components/misc/alert';
```

Reply:
Also, please don't reorder any of the exports.

Reply:
i made them alphabetical order :(